Problem using TinyUSB library with platformio

Hello,

First I'm sorry if that's an issue with platformio and not arduino-pico. But this was working before I switched to arduino-pico core and I can't find a way to make it work since.

Here is my `platformio.ini`file :
```
[env:pico]
platform = https://github.com/maxgerhardt/platform-raspberrypi.git
framework = arduino
board = pico
board_build.core = earlephilhower
board_build.filesystem_size = 0.5m
platform_packages =
    maxgerhardt/framework-arduinopico@https://github.com/earlephilhower/arduino-pico.git
    maxgerhardt/toolchain-pico@https://github.com/earlephilhower/pico-quick-toolchain/releases/download/1.3.4-d/x86_64-apple-darwin14.arm-none-eabi-46ea946.220608.tar.gz
lib_compat_mode = off
lib_deps = 
	fortyseveneffects/MIDI Library@^5.0.2
	adafruit/Adafruit MPR121@^1.1.1
	adafruit/Adafruit TinyUSB Library@^1.10.3
```

I got the following issue when trying to build : 
```

Processing pico (platform: https://github.com/maxgerhardt/platform-raspberrypi.git; framework: arduino; board: pico)
---------------------------------------------------------------------------------------------------------------------------------
Verbose mode can be enabled via `-v, --verbose` option
CONFIGURATION: https://docs.platformio.org/page/boards/raspberrypi/pico.html
PLATFORM: Raspberry Pi RP2040 (1.7.0+sha.8d7a063) > Raspberry Pi Pico
HARDWARE: RP2040 133MHz, 264KB RAM, 2MB Flash
DEBUG: Current (cmsis-dap) External (cmsis-dap, jlink, picoprobe, raspberrypi-swd)
PACKAGES: 
 - framework-arduinopico @ 1.20003.0+sha.3fb8cbf 
 - tool-rp2040tools @ 1.0.2 
 - toolchain-pico @ 5.100300.220608 (10.3.0) 
 - toolchain-rp2040-earlephilhower @ 5.100300.220608 (10.3.0)
Flash size: 2.00MB
Sketch size: 1.50MB
Filesystem size: 0.50MB
Maximium Sketch size: 1568768 EEPROM start: 0x101ff000 Filesystem start: 0x1017f000 Filesystem end: 0x101ff000
LDF: Library Dependency Finder -> https://bit.ly/configure-pio-ldf
LDF Modes: Finder ~ chain, Compatibility ~ off
Found 36 compatible libraries
Scanning dependencies...
Dependency Graph
|-- MIDI Library @ 5.0.2
|-- Adafruit MPR121 @ 1.1.1
|   |-- Adafruit BusIO @ 1.11.6
|   |   |-- Wire @ 1.0
|   |   |-- SPI @ 1.0
|   |-- Wire @ 1.0
|   |-- SPI @ 1.0
|-- Wire @ 1.0
Building in release mode

[...]

src/main.cpp:3:10: fatal error: Adafruit_TinyUSB.h: No such file or directory

**************************************************************************
* Looking for Adafruit_TinyUSB.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:Adafruit_TinyUSB.h"
* Web  > https://registry.platformio.org/search?q=header:Adafruit_TinyUSB.h
*
**************************************************************************

    3 | #include <Adafruit_TinyUSB.h>
      |          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pio/build/pico/src/main.cpp.o] Error 1
================================================== [FAILED] Took 3.76 seconds ==================================================
```

As you can see TinyUSB is not in the dependency graph and then header file can not be found.
I thought it was an issue with platformio thinking it was not compatible with pico board so I added `lib_compat_mode = off` in my platformio.ini file but it does not change anything.
Using platformio default pico config, this worked well and I could include and use TinyUSB.

Thank you very much for your help.

--- a/tools/platformio-build.py
+++ b/tools/platformio-build.py
@@ -159,6 +159,9 @@ def configure_usb_flags(cpp_defines):
     if "USE_TINYUSB" in cpp_defines:
         env.Append(CPPPATH=[os.path.join(
             FRAMEWORK_DIR, "libraries", "Adafruit_TinyUSB_Arduino", "src", "arduino")])
+        # automatically build with lib_archive = no to make weak linking work, needed for TinyUSB
+        env_section = "env:" + env["PIOENV"]
+        platform.config.set(env_section, "lib_archive", False)
     elif "PIO_FRAMEWORK_ARDUINO_NO_USB" in cpp_defines:
         env.Append(
             CPPPATH=[os.path.join(FRAMEWORK_DIR, "tools", "libpico")],

